libbuteosyncfw/profile/SyncSchedule.h

```diff
@@ -38,10 +38,8 @@ class SyncScheduleTest;
 typedef QSet<int> DaySet;
 
 const QString SYNC_SCHEDULE_ENABLED_KEY_BOOL("scheduler/schedule_enabled");
-const QString SYNC_EXTERNAL_SCHEDULE_ENABLED_KEY_BOOL("scheduler/external_schedule_enabled");
```

Comment:
Not used atm, and introduced by previous commit on same topic, no need to have them for now.